Automattic/wp-calypso
# Stats: Updating page header for summary details

#### Proposed Changes

* Updating the header for a summary details page (behind a feature flag)
* Adding a mixin file combining CSS needed by the summary details page and:
    * Section heading (used above the chart on the Traffic page)
    * Navigation segments (used above the chart on the Traffic page)

This PR continues work started in #71822 

#### Testing Instructions



* Navigate to `Stats` and make sure that the header above the chart on the Traffic page didn't change (consider multiple breakpoints)
* Navigate to `View all`/`View deatils` for stats modules below the chart (summary detail page) and make sure that horizontal bars are NOT visible and the original header is visible
* apply the feature flag by adding `?flags=stats/horizontal-bars-everywhere` to the URL
* confirm that the summary detail pages now show horizontal bars and a new header, similar to the one above the chart on the main page (period buttons should not have an equal width here)
* confirm that the Insights page loads without problems (The tags component is a work in progress)

Before applying a feature flag
![Screenshot 2023-01-12 at 11 36 17 AM](https://user-images.githubusercontent.com/112354940/211934790-b6517c8e-f9fe-4b90-8a22-4d29e14b6a2a.png)

Header with a feature flag not including changes in this PR
![Screenshot 2023-01-12 at 11 37 23 AM](https://user-images.githubusercontent.com/112354940/211934883-4d4dd98b-f20b-4561-9f68-e923fce611ee.png)

Header with a feature flag and proposed changes
![Screenshot 2023-01-12 at 11 37 38 AM](https://user-images.githubusercontent.com/112354940/211934964-70ee4a34-c34a-4da6-aa81-d79192a11b7e.png)


#### Pre-merge Checklist





Related to #71638
